Assets/Scripts/UI/Control/Cursor.cs

```diff
@@ -194,16 +195,18 @@ void UpdatePieceLocation()
         }
     }
 
-    void UpdateTimeline() 
-    {
-        selectedUnit = board.GetTile(currCoords).unit;
-        if (selectedUnit != null && player.IsOwner(selectedUnit))
-        {
-            uiManager.DisplayTimelineIcons(selectedUnit.plan);
-        }
-        else
-        {
-            uiManager.DisplayTimelineIcons(null);
-        }
-    }
+	void UpdateTileInfo()
```

Comment:
Just being picky, but what's the reason why this method is named `UpdateTileInfo()`?

Reply:
It's a misleading name now that you mention it. It was supposed to mean "update info after hovering over a new tile". I'll change it to `UpdateUnitInfo()`.